src/py_scripts/fc_fetch_reads.py

```diff
@@ -0,0 +1,3 @@
+from falcon_kit.mains.fetch_reads import main
```

Comment:
ok. I am a little behind on this one. I will take a look too so I can make the FALCON-Unzip refactoring to conform with the new principle.